src/store/profile/sagas.ts

```diff
@@ -151,15 +150,18 @@ export function* changeRoleSaga(action: ChangeRoleAction) {
 
 export function* getProfileSaga() {
   try {
-    const profile: GetProfileResponse = yield call(request, {
+    const profileDetails: GetProfileResponse = yield call(request, {
```

Comment:
Why did this have to change to something more verbose? It is implicit that any object such as a 'profile' is something that is composed of multiple properties, aka details.